db_querys.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+
+require 'dbConn.php';
+
+
+//setting the data into the  about me page
+function set_about_me($content, $picture_link, $db, $email, $phone_number)
+{
+        $query = $db->prepare('UPDATE `about_me` SET content=?, picture_link=?, phone_number=?, email=? WHERE `id` = 1;');
+        $query->bindParam(1, $content);
+        $query->bindParam(2, $picture_link);
+        $query->bindParam(3, $phone_number);
+        $query->bindParam(4, $email);
+        $query->execute();
+        $row_count = $query->rowCount();
+        print("Replaced $row_count row(s). \n");
+}
+
+
+
+$content = $_POST[content];
```

Comment:
Lines 22 - 25 are not validated in any way.  Consider a function to check if the array keys exist and if they are not empty.

need to put '' around the array key

Reply:
nice nice